### ISSUE TITLE
failing to build on macos

### ISSUE THREAD

**Issue by venus-as-a-boy** [mlemerre/l-lang#5]
I can give any extra info you may want. Replaced mentions of my path wit `~`.

--- Checking for ocamlfind... (found ~/.opam/system/bin/ocamlfind)
--- Checking for ocamlc.opt... (found /usr/local/bin/ocamlc.opt)
--- Checking for ocamlopt.opt... (found /usr/local/bin/ocamlopt.opt)
--- Checking for ocamldep.opt... (found /usr/local/bin/ocamldep.opt)
--- Checking for ocamllex.opt... (found /usr/local/bin/ocamllex.opt)
--- Checking whether ocamlc understands the "z" warnings... (yes)
--- Checking whether ocamlopt can create cmxs plugins... (yes)
File OMakefile: line 75, characters 26-55:
   Warning: old-style foreach expression.
   This expression should use a => binding.
File OMakefile: line 86, characters 82-108:
   Warning: old-style foreach expression.
   This expression should use a => binding.
*** omake error:
   File OMakefile: line 90, characters 1-46
   directory does not exist: ~/l-lang/build/next

**Comment by markjreed** [comment]
Yup, seeing the same thing. Why is it trying to build `next`? The top-level directory doesn't even exist in the repo, yet the OMakefile references some files that should be in it?
